java-11/Dockerfile

```diff
@@ -32,7 +32,9 @@ jdk.crypto.ec,\
 java.security.jgss,\
 jdk.security.auth,\
 # sun.misc.Unsafe and friends
-jdk.unsupported\
+jdk.unsupported,\
+# jdk specific network options
+jdk.net\
```

Comment:
big increase in image size?

Reply:
Actually, I don't see any increase in the image size at all.

**Previously:**
```
hypertrace/java                                                     11                                               4bbacc3c72ef        2 weeks ago         102MB
```

**Now:**
```
hypertrace/java                                                     11                                               5240110aebd4        5 seconds ago       102MB
```

Reply:
Excellent!